x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/cards/antivirus_registration_card.tsx

```diff
@@ -26,36 +28,51 @@ const DESCRIPTION = i18n.translate(
   'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.explanation',
   {
     defaultMessage:
-      'Toggle on to register Elastic as an official Antivirus solution for Windows OS. ' +
+      'Enable to register Elastic as an official Antivirus solution for Windows OS. ' +
       'This will also disable Windows Defender.',
   }
 );
 
-export const LABEL = i18n.translate(
-  'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.type',
-  {
-    defaultMessage: 'Register as antivirus',
-  }
-);
+export const LABELS: Record<AntivirusRegistrationModes, string> = {
+  [AntivirusRegistrationModes.disabled]: i18n.translate(
+    'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.disabled',
+    { defaultMessage: 'Disabled' }
+  ),
+  [AntivirusRegistrationModes.enabled]: i18n.translate(
+    'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.enabled',
+    { defaultMessage: 'Enabled' }
+  ),
+  [AntivirusRegistrationModes.sync]: i18n.translate(
+    'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.syncWithMalwarePrevent',
+    { defaultMessage: 'Sync with Malware Prevent' }
```

Comment:
```suggestion
    { defaultMessage: 'Sync with Malware Protection level' }
```
Either way, I believe we could use a popover of sorts to explain what will happen on `Prevent` and `Detect`

Reply:
done, also added hint 3f23c6d292e696cece2061a90a1fb4f786c8270d